src/support/steps/ll-css-bg-image.ts

```diff
@@ -58,4 +58,113 @@ Then('I must see the correct style in the head', async function (this: ICustomWo
   console.log(failMessage);
 
   expect(isMatch, failMessage).toBeTruthy();
+});
+
+const LL_BACKGROUND_IMAGES = {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    lazyload_css_background_images: {
+        initialImages: [
+            'https://new.rocketlabsqa.ovh/wp-content/rocket-test-data/images/test.png',
```

Comment:
For images with e2e domain, can we read this from env variable so it can run on any e2e env? what do you think?